docs/source/intervention_models/crm_mgmt/scenarios.rst

```diff
@@ -0,0 +1,64 @@
+Scenarios
+*********
+
+Outreach
+~~~~~~~~~~~~~~
+Under this scenario, the effectiveness of the delivery of SBP-lowering and LDL-c lowering therapies would increase through methods such as a phone call, app, or support clinic. 
```

Comment:
as we discussed today, I would like the scenarios.rst to describe what the coverage of the 3 interventions (outreach, lifestyle, polypill) is at each time in each scenario.  A lot of the background info about each specific intervention can go into the corresponding intervention doc.

Reply:
have made edits to the outreach section; will follow up with additional edits to the other two interventions.